### PR TITLE
Feat/ping route

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: stores/geode.js [hunk 1]
```diff
@@ -25,16 +42,40 @@ export const use_geode_store = defineStore("geode", {
...
+          },
+          response_function: () => {
+            this.is_running = true
+          },
+          response_error_function: () => {
+            feedback_store.$patch({ server_error: true })
+          },
+        },
+      )
+      // return new Promise((resolve, reject) => {
```
Comment (BotellaA) [inline stores/geode.js:59]: To remove or to keep?

File: stores/infra.js [hunk 2]
```diff
@@ -68,8 +58,6 @@ export const use_infra_store = defineStore("infra", {
       if (isElectron()) {
         await window.electronAPI.run_back(geode_store.PORT)
```
Comment (BotellaA) [inline stores/infra.js:59]: Electron should not be used in ogweb?

File: stores/viewer.js [hunk 3]
```diff
@@ -15,13 +14,29 @@ export const use_viewer_store = defineStore("viewer", {
...
+      if (use_infra_store().is_cloud) {
+        return "443"
+      } else {
+        return this.default_local_port
+      }
+    },
+    base_url() {
       const infra_store = use_infra_store()
-      var viewer_url = `${state.PROTOCOL}://${infra_store.domain_name}:${state.PORT}`
+      var viewer_url = `${this.protocol}://${infra_store.domain_name}:${this.port}`
```
Comment (BotellaA) [inline stores/viewer.js:33]: Let or var?
this.port or port()? Same question in the other store and for protocol

File: stores/geode.js (hunk 1 above)
Comment (JulienChampagnol) [reply stores/geode.js:59]: Still a wip

File: stores/infra.js (hunk 2 above)
Comment (JulienChampagnol) [reply stores/infra.js:59]: Only if is isElectron returns true, I thought we agreed on that

File: stores/viewer.js (hunk 3 above)
Comment (JulienChampagnol) [reply stores/viewer.js:33]: Indeed for var, getters aren't functions, so no parenthesis

File: stores/infra.js (hunk 2 above)
Comment (BotellaA) [reply stores/infra.js:59]: Yes, I missed the isElectron..

**Pull request merged**